takwimu/templates/profile/profile_detail_base.html

```diff
@@ -157,6 +157,7 @@
                   linkedin: '{{ settings.takwimu.SocialMediaSetting.linkedin|safe }}'
               }
           };
+      takwimu.data_topic_page_description = '{{ data_by_topic_description|safe }}';
```

Comment:
there is already a HURUmap variable `profileData` defined. I think we can kill two birds with one stone by checking for that variable and adding it there. E.g.
```javascript

profileData = profileData || {}
profileData.description = ....
```

This way, we wouldn't need to do anything in the front end as `profileData` is already a known variable there: https://github.com/TakwimuAfrica/Takwimu/blob/570cecc2385ff9f7b14fd733e93e2996e1a04b83/takwimu/takwimu_ui/src/index.js#L25